client/secureclient/src/main/java/com/linkedin/openhouse/client/ssl/TablesApiClientFactory.java

```diff
@@ -50,6 +50,7 @@ public ApiClient createApiClient(@NonNull String baseUrl, String token, String t
       throws MalformedURLException, SSLException {
     WebClient webClient = createWebClient(baseUrl, token, truststoreLocation);
     ApiClient apiClient = new ApiClient(webClient);
+    apiClient.addDefaultHeader("Authorization", String.format("Bearer %s", token));
```

Comment:
Is there an existing constant for the Header that can be used in the reactor netty lib?